plugins/GSdx/res/convert.fx

```diff
@@ -219,6 +219,141 @@ PS_OUTPUT ps_main9(PS_INPUT input) // triangular
 	return output;
 }
 
+// DUMMY SHADERS START
+PS_OUTPUT ps_main10(PS_INPUT input)
+{
+	PS_OUTPUT output;
+
+	output.c = 0;
+
+	return output;
+}
+
+PS_OUTPUT ps_main11(PS_INPUT input)
+{
+	PS_OUTPUT output;
+
+	output.c = 0;
+
+	return output;
+}
+
+PS_OUTPUT ps_main12(PS_INPUT input)
+{
+	PS_OUTPUT output;
+
+	output.c = 0;
+
+	return output;
+}
+
+PS_OUTPUT ps_main13(PS_INPUT input)
+{
+	PS_OUTPUT output;
+
+	output.c = 0;
+
+	return output;
+}
+
+PS_OUTPUT ps_main14(PS_INPUT input)
+{
+	PS_OUTPUT output;
+
+	output.c = 0;
+
+	return output;
+}
+
+PS_OUTPUT ps_main15(PS_INPUT input)
+{
+	PS_OUTPUT output;
+
+	output.c = 0;
+
+	return output;
+}
+
+PS_OUTPUT ps_main16(PS_INPUT input)
+{
+	PS_OUTPUT output;
+
+	output.c = 0;
+
+	return output;
+}
+// DUMMY SHADERS END
+
+PS_OUTPUT ps_main17(PS_INPUT input)
+{
+    PS_OUTPUT output;
+ 
+    // Potential speed optimization. There is a high probability that
+    // game only want to extract a single channel (blue). It will allow
+    // to remove most of the conditional operation and yield a +2/3 fps
+    // boost on MGS3
+    //
+    // Hypothesis wrong in Prince of Persia ... Seriously WTF !
+    //#define ONLY_BLUE;
+ 
+    // Convert a RGBA texture into a 8 bits packed texture
+    // Input column: 8x2 RGBA pixels
+    // 0: 8 RGBA
+    // 1: 8 RGBA
+    // Output column: 16x4 Index pixels
+    // 0: 8 R | 8 B
+    // 1: 8 R | 8 B
+    // 2: 8 G | 8 A
+    // 3: 8 G | 8 A
+    float c;
+ 
+    uint2 sel = uint2(input.p.xy) % uint2(16u, 16u);
+    int2  tb  = ((int2(input.p.xy) & ~int2(15, 3)) >> 1);
+ 
+    int ty   = tb.y | (int(input.p.y) & 1);
+    int txN  = tb.x | (int(input.p.x) & 7);
+    int txH  = tb.x | ((int(input.p.x) + 4) & 7);
+ 
+    // TODO investigate texture gather
+    float4 cN = Texture.Sample(TextureSampler, int2(txN, ty), 0);
+    float4 cH = Texture.Sample(TextureSampler, int2(txH, ty), 0);
```

Comment:
I don't think `texelFetch` translates to `Sample`. I think `Sample` uses normalized device coordinates but you've provided it with screen (window) space coordinates instead. Maybe try `Load` instead? 

https://docs.microsoft.com/en-us/windows/desktop/direct3dhlsl/dx-graphics-hlsl-to-load